.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 4,
+  "semi": true,
+  "singleQuote": true
+}
```

Comment:
maybe it would be good to also have a prettier run on every commit?
https://prettier.io/docs/en/precommit.html

Reply:
We are moving project to React (take a look at https://github.com/openstreetmap-polska/ua-2022-map-v2) so it is pointless to configure it right there.